frontend/src/Pages/RecruitmentApplicationFormPage/RecruitmentApplicationFormPage.tsx

```diff
@@ -240,35 +279,61 @@ export function RecruitmentApplicationFormPage() {
           </div>
-          {similarPositionsBtns}
-          {otherPositionsAtGang}
+
+          {!isMobile && (
+            <div className={styles.other_positions}>
+              {similarPositionsBtns}
+              {otherPositionsAtGang}
+            </div>
+          )}
         </div>
-        {recruitmentApplication && (
+        {recruitmentApplication?.withdrawn && (
           <div className={styles.withdrawn_container}>
-            {recruitmentApplication?.withdrawn ? (
-              <Text size="l" as="i" className={styles.withdrawn_text}>
-                {t(KEY.recruitment_withdrawn_message)}:
-              </Text>
-            ) : (
-              <Button theme="samf" display="basic" onClick={() => withdrawApplication()}>
-                {t(KEY.recruitment_withdraw_application)}
-              </Button>
-            )}
+            <Text size="l" as="i" className={styles.withdrawn_text}>
+              {t(KEY.recruitment_withdrawn_message)}
+            </Text>
           </div>
         )}
         {user ? (
-          <SamfForm
-            initialData={recruitmentApplication as FormProps}
-            onSubmit={handleOnSubmit}
-            submitText={submitText}
-            devMode={false}
-          >
-            <h2 className={styles.label}>{t(KEY.recruitment_application)}:</h2>
-            <SamfFormField field="application_text" type="text_long" />{' '}
-          </SamfForm>
+          <>
+            <Form {...form}>
+              <form onSubmit={form.handleSubmit(handleOnSubmit)}>
+                <div className={styles.form_header}>
+                  <h2 className={styles.label}>{t(KEY.recruitment_application)}</h2>
+                  <ToolTip value={t(KEY.recruitment_applyforhelp)}>
+                    <Icon icon="mingcute:question-fill" width="1.2em" height="1.2em" />
+                  </ToolTip>
+                </div>
+                <FormField
+                  control={form.control}
+                  name="application_text"
+                  render={({ field }) => {
+                    return (
+                      <FormItem>
+                        <FormControl>
+                          <Textarea
+                            {...field}
+                            value={field.value} // Ensure the Textarea is controlled
+                          />
+                        </FormControl>
+                      </FormItem>
+                    );
+                  }}
+                />
+                <div className={styles.form_buttons}>
+                  <Button type="submit" theme="green" display="basic">
+                    {submitText}
+                  </Button>
+                  {!recruitmentApplication?.withdrawn && (
+                    <Button type="button" theme="samf" display="basic" onClick={() => withdrawApplication()}>
+                      {t(KEY.recruitment_withdraw_application)}
+                    </Button>
+                  )}
```

Comment:
Viser bare knappen hvis man har søkt, men ikke hvis søknaden er trukket

```suggestion
                  {!recruitmentApplication?.withdrawn && recruitmentApplication && (
                    <Button type="button" theme="samf" display="basic" onClick={() => withdrawApplication()}>
                      {t(KEY.recruitment_withdraw_application)}
                    </Button>
                  )}
```